### PR TITLE
[10.0] [ADD] hr_payslip_change_state

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
HR Payslip Change State
=======================

This module process payslip batches:

* This module allows to mass change Payslip states
* The module checks if the required action is allowed for each payslip
* If any of the payslips are not in a suitable state a warning message will  pop up and no changes will be made

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: hr_payslip_change_state/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": "Human Resources Payslip Change State",
     "summary": "Change the state of many payslips at a time",
     "version": "10.0.1.0.0",
     "license": "AGPL-3",
     "category": "Human Resources",
-    "author": "Eficent, Odoo Community Association (OCA)",
+    "author": "Eficent, Odoo Community Association (OCA)"
+              "Serpent Consulting Services Pvt. Ltd.,",
```
Comment (LoisRForgeFlow) [inline hr_payslip_change_state/__manifest__.py]: OCA must be always the last one. But anyway I wouldn't put Serpent as author as what you did here is just a standard migration, and you didn't add any new feature.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by LoisRForgeFlow** [comment]
And please, add always the dependencies to other PR or modules not available in OCA.

Depends on https://github.com/OCA/hr/pull/355

**Comment by nikul-serpentcs** [comment]
@aheficent @lreficent  Could you please review.

**Comment by AaronHForgeFlow** [comment]
runbot is down

**Comment by nikul-serpentcs** [comment]
@aheficent Yes, because dependency module `hr_payroll_cancel` is not available.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs hr_payroll_cancel is available now, would you rebase?

**Review by AaronHForgeFlow — COMMENTED** [review]
Something is wrong here, 98 commits and 461 files changed.

**Comment by nikul-serpentcs** [comment]
It is done now! I checked the commits are already rebased to the OCA/hr 10.0

**Comment by feketemihai** [comment]
@nikul-serpentcs I made a PR, https://github.com/OCA/hr/pull/435 which will solve your travis error, please include that commit, or wait to rebase after is merged.

**Comment by MiquelRForgeFlow** [comment]
Superseeded by https://github.com/OCA/hr/pull/462. This should be closed.

**Pull request closed without merging**